Hazel/Crypto/AesGcm.cs

```diff
@@ -322,48 +322,59 @@ static void MultiplyGF128Elements(ByteSpan X, ByteSpan Y, ByteSpan scratchZ, Byt
             //   irreducible polynomial is represented by the bit string:
             //   `11100001` followed by 120 `0`s. We can add this value to `V`
             //   by: `V[0] = V[0] ^ 0xE1`.
-            SetSpanToZeros(scratchZ);
+            scratchZ.SecureClear();
             X.CopyTo(scratchV);
 
-            for (int ii = 0; ii != 128; ++ii)
+            unchecked
```

Comment:
Nope, you're right! I'm going to leave it because it's kind of a clarity thing, but "non-constant expressions" are by default unchecked.

https://learn.microsoft.com/en-us/dotnet/csharp/language-reference/language-specification/expressions#12820-default-value-expressions